packages/map-template/src/components/Search/Search.scss

```
@@ -64,7 +64,13 @@
         background-color: white;
     }
 
+    @keyframes askAiSlideIn {
```

Stylelint (17.10.0)

[error] Expected keyframe name "askAiSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)